src/System.Private.CoreLib/shared/System/MemoryExtensions.Fast.cs

```diff
@@ -219,11 +219,14 @@ public static int LastIndexOf(this ReadOnlySpan<char> span, ReadOnlySpan<char> v
         /// <param name="source">The source span.</param>
         /// <param name="destination">The destination span which contains the transformed characters.</param>
         /// <param name="culture">An object that supplies culture-specific casing rules.</param>
-        /// <remarks>If the source and destinations overlap, this method behaves as if the original values are in
-        /// a temporary location before the destination is overwritten. If <paramref name="culture"/> is null, <see cref="System.Globalization.CultureInfo.CurrentCulture"/> will be used.</remarks>
+        /// <remarks>If the source and destinations overlap, this method throws InvalidOperationException. 
```

Comment:
Use the exception tag. Same elsewhere.

```suggestion
        /// <exception cref="InvalidOperationException">
        ///   The source and destination buffers overlap.
        /// </exception>
```

Reply:
Added exception tag in one line is it ok?

Reply:
Yep.